packages/plugin-user/src/user.js

```diff
@@ -164,6 +164,31 @@ const User = SparkPlugin.extend({
   },
 
   /**
+   * Extracts a user object for a user specified by userId(UUID) and organization specified by orgId(UUID)
+   * @param {Object} options
+   * @param {string} options.userId
+   * @param {string} options.orgId
+   * @private
+   * @returns {Promise<Object>}
+   */
+  getUserForOrg(options) {
```

Comment:
I'm not entirely comfortable using CI apis that require `orgId` knowledge. What user details do you need? Should we use hybrid services as motivation to build a proper user details endpoint?

@pcl, do you have an opinion on clients talking direct to CI for user info lookups?